packages/tools/pie-monorepo-utils/component-statuses/generate-component-statuses.js

```diff
@@ -8,4 +8,4 @@ const generator = new ComponentStatusGenerator({
     fs, path, promisify, console,
 });
 
-generator.generateStatuses(process.cwd());
```

Comment:
Just curious if there was a specific reason you put this back in? I think I originally wrote it to be more testable like this (but then it looks like I didn't write any tests 🤦🏻‍♂️)

Reply:
I _think_ this was probably done because originally, this script could be invoked via `npx`.

However, we shouldn't need to pass this in anymore, as the `pie-monorepo-utils` package itself is responsible for calling this script.